DashAI/back/api/api_v1/endpoints/runs.py

```diff
@@ -0,0 +1,207 @@
+import logging
+from typing import Union
+
+from fastapi import APIRouter, Depends, Response, status
+from fastapi.exceptions import HTTPException
+from sqlalchemy import exc, select
+from sqlalchemy.orm import Session
+
+from DashAI.back.api.deps import get_db
+from DashAI.back.database.models import Experiment, Run
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("/")
+async def get_run(run_id: int, db: Session = Depends(get_db)):
+    """
+    Returns the run with the spécified id.
+
+    Returns
+    -------
+    JSON
+        run JSON
+    """
+
+    try:
+        run = db.get(Run, run_id)
+        if not run:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="Run not found",
+            )
+    except exc.SQLAlchemyError as e:
+        log.exception(e)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Internal database error",
+        )
+    return run
+
+
+@router.get("/list/{experiment_id}")
+async def get_run_from_experiment(experiment_id: int, db: Session = Depends(get_db)):
+    """
+    Returns the runs associated with the specified experiment_id from the database.
+
+    Parameters
+    ----------
+    experiment_id : int
+        id of the experiment assoc with the runs to query.
+
+    Returns
+    -------
+    List[JSON]
+        List of run JSON associated with the specified experiment id.
+    """
+    try:
+        runs = db.scalars(select(Run).where(Run.experiment_id == experiment_id)).all()
+        if not runs:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="Runs assoc with Experiment not found",
+            )
+    except exc.SQLAlchemyError as e:
+        log.exception(e)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Internal database error",
+        )
+    return runs
+
+
+@router.post("/", status_code=status.HTTP_201_CREATED)
+async def upload_run(
+    experiment_id: int,
+    model_name: str,
+    name: str,
+    parameters: dict,
+    description: str = "",
```

Comment:
If empty, then shouldn't the default value be None?
(idk what happens with this change in the db).

```suggestion
    description: str = None,
```